### PR TITLE
Add intro to errors reference

### PR DESCRIPTION
**Pull request by lorensr**
Adds info about what errors are listed, and where to look for different types of errors.

### CODE REVIEW

File: assembly/guide-configs/errors.json [hunk 1]
```diff
@@ -4,10 +4,14 @@
   "file_dir": "references",
   "title": "Errors",
   "sidebar_label": "Errors",
-  "description": "This guide is an overview of the errors that can appear on the Temporal Web UI.",
```
Comment (lorensr) [inline assembly/guide-configs/errors.json:7]: There are other errors that appear in the UI, including different history errors and non-history errors like no worker running

File: assembly/workflows/fullAssembly.js [hunk 2]
```diff
@@ -7,13 +7,19 @@ const simpleActivityOptions = {
...
+  });
+
   const config = await activities.getConfig(params);
 
   await activities.createTempDir(config);
 
   await activities.genSourceObjects(config);
 
-  await activities.attachSourceToGuides(config);
+  await deterministicActivities.attachSourceToGuides(config);
```
Comment (flossypurse) [inline assembly/workflows/fullAssembly.js:22]: Nice :)

**Review by DJSanti — APPROVED** [review]

File: docs-src/references/strongly-typed-errors/intro.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+id: intro
+---
+
+This reference lists possible [Workflow Task](/tasks#workflow-task) errors and how to resolve them.
```
Comment (DJSanti) [inline docs-src/references/strongly-typed-errors/intro.md:5]:
```suggestion
This reference lists possible [Workflow Task](/tasks/#workflow-task) errors and how to resolve them.
```

File: docs/references/errors.md [hunk 4]
```diff
@@ -11,6 +11,12 @@ toc_max_heading_level: 4
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+This reference lists possible [Workflow Task](/tasks#workflow-task) errors and how to resolve them. 
+
+> For other types of errors, see [Temporal Failures](https://docs.temporal.io/kb/failures).
+
+Each of the below errors corresponds with a [WorkflowTaskFailedCause](https://api-docs.temporal.io/#temporal.api.enums.v1.WorkflowTaskFailedCause), which appears in [Events](/workflows#event) under `workflow_task_failed_event_attributes`.
```
Comment (DJSanti) [inline docs/references/errors.md:18]:
```suggestion
Each of the below errors corresponds with a [WorkflowTaskFailedCause](https://api-docs.temporal.io/#temporal.api.enums.v1.WorkflowTaskFailedCause), which appears in [Events](/workflows/#event) under `workflow_task_failed_event_attributes`.
```

**Review by DJSanti — CHANGES_REQUESTED** [review]
Some minor link fixes. Need slashes before the pound signs.

File: docs-src/references/strongly-typed-errors/intro.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+> For other types of errors, see [Temporal Failures](https://docs.temporal.io/kb/failures).
+
+Each of the below errors corresponds with a [WorkflowTaskFailedCause](https://api-docs.temporal.io/#temporal.api.enums.v1.WorkflowTaskFailedCause), which appears in [Events](/workflows#event) under `workflow_task_failed_event_attributes`.
```
Comment (DJSanti) [inline docs-src/references/strongly-typed-errors/intro.md:9]:
```suggestion
Each of the below errors corresponds with a [WorkflowTaskFailedCause](https://api-docs.temporal.io/#temporal.api.enums.v1.WorkflowTaskFailedCause), which appears in [Events](/workflows/#event) under `workflow_task_failed_event_attributes`.
```

**Review by DJSanti — CHANGES_REQUESTED** [review]
Some minor link fixes. Need slashes before the pound signs.

File: docs-src/references/strongly-typed-errors/intro.md (hunk 3 above)
Comment (flossypurse) [reply docs-src/references/strongly-typed-errors/intro.md:5]: Actually this should link to `/concepts/what-is-a-workflow-task` and link magic auto rewrites to this.

But `/` before `#` doesn't work any more -

**Review by DJSanti — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**